web-api/domain-s3-restore.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
```

Comment:
@ericsorenson Could we get a description for the purpose of this script?

Reply:
It's a one time script for this PR, documented in https://github.com/flexion/ef-cms/issues/5666#issuecomment-667339500